chromadb/migrations/metadb/00002-embedding-metadata.sqlite.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE embedding_metadata_lists (
+    id INTEGER REFERENCES embeddings(id),
+    key TEXT NOT NULL,
+    string_value TEXT,
+    float_value REAL,
+    int_value INTEGER,
+    FOREIGN KEY (id, key) REFERENCES embedding_metadata(id, key) ON DELETE CASCADE
```

Comment:
I couldn't get this delete cascade to work. So I manually delete them similar to how `embedding_metadata` is deleted.

Should I remove the `ON DELETE` or spend some time figuring it out?